DuckDuckGo/TabViewController.swift

```diff
@@ -318,13 +307,6 @@ class TabViewController: UIViewController {
         webView.addObserver(self, forKeyPath: #keyPath(WKWebView.canGoForward), options: .new, context: nil)
         webView.addObserver(self, forKeyPath: #keyPath(WKWebView.title), options: .new, context: nil)
     }
-    
-    private func attachLongPressHandler(webView: WKWebView) {
-        let gestrueRecognizer = UILongPressGestureRecognizer(target: self, action: #selector(onLongPress(sender:)))
```

Comment:
I'm gonna remove the `onLongPress(sender:)` function too, looks like this was the only place using it